app/app-config.yaml

```diff
@@ -64,21 +64,30 @@ integrations:
         - clientId: ${AUTH_MICROSOFT_CLIENT_ID}
           clientSecret: ${AUTH_MICROSOFT_CLIENT_SECRET}
           tenantId: ${AUTH_MICROSOFT_TENANT_ID}
-          
+
   github:
     - host: github.com
       apps:
         - $include: github-app-configuration.yaml
 
+rbac:
+  platformAdminsGroup: ${ADP_PORTAL_PLATFORM_ADMINS_GROUP}
```

Comment:
These variables need to be added to build.yaml and backstage.yaml. Also Please update variable group adp-portal-ssv5

Reply:
I have added them